docs/pages/workflow/development-mode.mdx

```diff
@@ -6,19 +6,21 @@ sidebar_title: Development and production
 import Video from '~/components/plugins/Video';
 import { Terminal } from '~/ui/components/Snippet';
 
-Your project will always run in one of two modes: development or production. By default, running your project locally with `expo start` runs it in development mode, whereas a published project (via `eas update`), or any standalone apps, will run in production mode.
+Your project will always run in two modes: **development** or **production**. By default, running your project locally with `npx expo start` runs it in development, whereas a published project (with `eas update`), or any standalone app, will run in production.
 
-Production mode [minifies your code](https://www.imperva.com/learn/performance/minification/) and better represents the performance your app will have on end users' devices. Development mode includes useful warnings and gives you access to tools that make development and debugging easier. Let's look at each of these modes more in detail and learn how you can toggle between them.
+**Development mode** includes useful warnings and gives you access to tools that make development and debugging easier. **Production mode** [minifies your code](https://www.imperva.com/learn/performance/minification/) and better represents the performance your app will have on end users' devices. Let's look at each of these modes more in detail and learn how you can toggle between them.
```

Comment:
You didn't change this, but I wonder why we link out to a commercial 3rd party here.

Reply:
No clue, but I'd replace that link with a little description minification in our docs: https://docs.expo.dev/guides/customizing-metro/#minification